tensorflow_addons/image/transform_test.py

```diff
@@ -22,6 +22,12 @@
 import tensorflow as tf
 
 from tensorflow.python.ops import gradient_checker
+from tensorflow.python.framework import dtypes
+from tensorflow.python import array_ops
```

Comment:
Unfortunately, I think those private APIs still exist. Please replace them by tf public API, as mentioned before.  Take zeros op as an example, we should use `tf.zeros`, rather than `array_ops.zeros`. That's what we did in https://github.com/tensorflow/addons/pull/55 , please take a look at it if useful.

Reply:
Done, with the exception of `name_scope` because the new public API is incompatible with the one currently used throughout our codebase (`ops.name_scope`).